src/mpi/comm/commutil.c

```diff
@@ -546,6 +550,17 @@ int MPII_Comm_is_node_consecutive(MPIR_Comm * comm)
 #undef FCNAME
 #define FCNAME "MPII_Comm_copy"
 int MPII_Comm_copy(MPIR_Comm * comm_ptr, int size, MPIR_Comm ** outcomm_ptr)
+{
+    return MPII_Comm_copy_with_info(comm_ptr, size, outcomm_ptr, NULL);
+}
+
+
+#undef FUNCNAME
+#define FUNCNAME MPII_Comm_copy_with_info
+#undef FCNAME
+#define FCNAME "MPII_Comm_copy_with_info"
+int MPII_Comm_copy_with_info(MPIR_Comm * comm_ptr, int size, MPIR_Comm ** outcomm_ptr,
```

Comment:
TBH, I'd rather just add an info parameter to `MPII_Comm_copy` than add this extra layer. What do you think @alexander-sannikov?

Reply:
@raffenet totally agree.